orbit/pkg/table/common/common.go

```diff
@@ -24,3 +27,28 @@ func GetConsoleUidGid() (uid uint32, gid uint32, err error) {
 	}
 	return stat.Uid, stat.Gid, nil
 }
+
+// GetValFromXMLWithTags Will look for a sequence of tags and will get the following nested value as string
+// In the following xml example the function will return "5" if called with parentTag = "someParentTag", tag = "someTag", tagValue = "someValue", valType = "integer"
+// <someParentTag>
+//   <someTag>someValue</someTag>
+//   <integer>5</integer>
+// </someParentTag>
+func GetValFromXMLWithTags(xml string, parentTag string, tag string, tagValue string, valType string) (maxFailedAttempts string, err error) {
+	doc, err := xmlquery.Parse(strings.NewReader(xml))
```

Comment:
Optimization nit: You should probably parse once and then call this method with the returned `doc` (instead of calling `xmlquery.Parse` four times on the same full document.)